bentoml/cli/resources.py

```diff
@@ -0,0 +1,228 @@
+    )
+    @click.option('--all-namespaces', is_flag=True)
+    @click.option(
+        '--limit', type=click.INT, help='Limit how many resources will be retrieved'
+    )
+    @click.option(
+        '--filters',
+        type=click.STRING,
+        help='List resources containing the filter string in name',
+    )
+    @click.option(
+        '-l',
+        '--labels',
+        type=click.STRING,
+        help='List deployments matching the giving labels',
+    )
+    @click.option('-o', '--output', type=click.Choice(['json', 'yaml', 'table']))
+    def deployment(name, namespace, all_namespaces, limit, filters, labels, output):
+        yatai_client = YataiClient()
+        if name:
+            track_cli('deploy-get')
+            output = output or 'json'
+            get_result = yatai_client.deployment.get(namespace, name)
+            if get_result.status.status_code != status_pb2.Status.OK:
+                error_code, error_message = parse_pb_response_error_message(
+                    get_result.status
+                )
+                _echo(
+                    f'Failed to get deployment {name}. '
+                    f'{error_code}:{error_message}',
+                    CLI_COLOR_ERROR,
+                )
+                return
+            describe_result = yatai_client.deployment.describe(
+                namespace=namespace, name=name
+            )
+            if describe_result.status.status_code != status_pb2.Status.OK:
+                error_code, error_message = parse_pb_response_error_message(
+                    describe_result.status
+                )
+                _echo(
+                    f'Failed to retrieve the latest status for Sagemaker deployment'
+                    f' {name}. {error_code}:{error_message}',
+                    CLI_COLOR_ERROR,
+                )
+                return
+            get_result.deployment.state.CopyFrom(describe_result.state)
+            _print_deployment_info(get_result.deployment, output)
+            return
+        else:
+            track_cli('deploy-list')
+            output = output or 'table'
+            list_result = yatai_client.deployment.list(
+                limit=limit,
+                filters=filters,
+                labels=labels,
+                namespace=namespace,
+                is_all_namespaces=all_namespaces,
+            )
+        if list_result.status.status_code != status_pb2.Status.OK:
+            error_code, error_message = parse_pb_response_error_message(
+                list_result.status
+            )
+            _echo(
+                f'Failed to list deployments. ' f'{error_code}:{error_message}',
+                CLI_COLOR_ERROR,
+            )
+        else:
+            _print_deployments_info(list_result.deployments, output)
+
+    @get_resource.command(help='Get BentoService information from db')
+    @click.option('-n', '--name', type=click.STRING, help='BentoService name')
```

Comment:
Can we use something like "bento tag" in deployments, to make them consistent?

```
bentoml get bento
bentoml get bento IrisClassifier
bentoml get bento IrisClassifier:20191126125258_4AB1D4

Reply:
if the "optional argument" is messy to implement with Click, let's consider the following instead?
```
bentoml list bento
bentoml get bento IrisClassifier
bentoml get bento IrisClassifier:20191126125258_4AB1D4
```

Reply:
yeah, let's just implement `list` command